packages/keystatic/src/form/fields/document/DocumentEditor/component-blocks/chromeful-element.tsx

```diff
@@ -190,38 +181,51 @@ function DefaultToolbarWithChrome({
 function FormValue({
   onClose,
   props,
-  isValid,
 }: {
   props: GenericPreviewProps<NonChildFieldComponentSchema, unknown>;
   onClose(): void;
-  isValid: boolean;
 }) {
+  const stringFormatter = useLocalizedStringFormatter(l10nMessages);
+  const formId = useId();
   const [forceValidation, setForceValidation] = useState(false);
+  const val: unknown[] = previewPropsToValue(props as any);
+  const [state, setState] = useState(val);
```

Comment:
```suggestion
  const [state, setState] = useState(() => previewPropsToValue(props as any) as unknown);
```

Just since it only needs to happen on mount